internal/compiler/tests/syntax/exports/single_global_missing_export.slint

```diff
@@ -0,0 +1,7 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+  global Settings {
+//^warning{Global singleton is implicitly marked for export. This is deprecated and it should be explicitly exported}
```

Comment:
Has this ever been the case?   It shouldn't be marked as export.

Reply:
Yes it currently is also exported :(. See the gallery change for example.